examples/golden_graph/node.cpp

```diff
@@ -1,3 +1,25 @@
 #include "node.hpp"
 
 Node::Node(int id, int value) : m_id {id}, m_value {value} {}
+
+int Node::getID() const
+{
+	return m_id;
+}
+
+int Node::getValue() const
+{
+	return m_value;
+}
+
+std::vector<Edges*>& Node::getEdges() const
```

Comment:
Add a const qualifier here too.
**const **std::vector<Edges*>&****

Reply:
Thanks for your attention, I will remove the omissions.